subworkflows/nf-core/fastq_align_chromap/main.nf

```diff
@@ -0,0 +1,37 @@
+/*
+ * Map reads, sort, index BAM file and run samtools stats, flagstat and idxstats
+ */
+
+include { CHROMAP_CHROMAP         } from '../../../modules/nf-core/chromap/chromap/main'
+include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
+
+workflow FASTQ_ALIGN_CHROMAP {
+    take:
+    reads // channel: [ val(meta), [ reads ] ]
+    index //    path: /path/to/index
+    fasta //    path: /path/to/fasta
```

Comment:
I've (we? ping @maxulysse ) been trying to push for adding meta values to reference data.
How would you feel about adding this?

Reply:
meta everrythiiiinnnngggggg

Reply:
this ⏫

Reply:
I almost implemented it... but was wondering about a real scenario where this information will be used in a pipeline, do you have an example?

Reply:
I'm working towards this in my workflows, since we can have data from multiple organisms on a sequencing runs
https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-preprocessing

Reply:
Fair enough! Thanks

Reply:
Doing this in eager: https://github.com/nf-core/eager/blob/71bb8a4069749f5bff74b6599fb18bad1450abb5/subworkflows/local/reference_indexing_multi.nf

And already implemented similar concept in taxprofiler (one tool, but multiple databases):  https://github.com/nf-core/taxprofiler/blob/dev/subworkflows/local/profiling.nf

Reply:
Implemented on a different PR #2416